frontend/src/metabase/admin/settings/components/SettingsSetting.jsx

```diff
@@ -28,6 +28,7 @@ const SETTING_WIDGET_MAP = {
   radio: SettingRadio,
   boolean: SettingToggle,
   text: SettingText,
+  hidden: () => null,
```

Comment:
In some cases, e.g. when we want to have an introduction, we could use `description: element` and `type: "hidden"`